server/modlog.ts

```diff
@@ -450,6 +458,13 @@ export class Modlog {
 		};
 	}
 
+	async destroy() {
+		if (this === Chat.modlog) {
```

Comment:
What's this guarding against?

Reply:
A modlog instance that's not the main one for the server (for example, one created by the converter) shouldn't set `Chat.modlogStreamPersistence` or destroy the ProcessManager since those are specific to `Chat.modlog`.

Reply:
Makes sense.

Reply:
Actually, is `Chat.modlogStreamPersistence` needed at all? It seems like reading `Chat.modlog.streams` and `Chat.modlog.sharedStreams` is enough, right?

Reply:
That would work; `Chat.modlogStreamPersistence` is just another level of abstraction.